Resources/Maps/Shuttles/DeltaV/listening_post.yml

```diff
@@ -339,11 +336,39 @@ entities:
           - 0
           - 0
           - 0
+        - volume: 2500
+          immutable: True
+          moles:
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
         chunkSize: 4
     - type: GasTileOverlay
     - type: RadiationGridResistance
     - type: Shuttle
     - type: NavMap
+  - uid: 29
+    components:
+    - type: MetaData
+      name: Map Entity
+    - type: Transform
+    - type: Map
```

Comment:
Did you save the map as grid? As in, did you use the cmd, `savegrid` and not `savemap`

Reply:
In my closed PR it was save gridded but it seems they recreated it

Reply:
rookie mistake -700 aura 😔